js/brushes/waterWave.js

```diff
@@ -0,0 +1,30 @@
+/* global mouseIsPressed, random, colorMode, fill, stroke, sin, ellipse,
+ HSB, frameCount, mouseX, mouseY */
+ var start = 0
```

Comment:
though the use of the `var` declaration technically works fine, it's better practice these days to use the `let` declaration anytime u're creating variables like these